docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+      "                         'diagnostics': 'Profile '\n",
+      "                                        'http://fhir.kids-first.io/StructureDefinition/Patient, '\n",
+      "                                        \"Element 'Patient.gender': minimum \"\n",
+      "                                        'required = 1, but only found 0',\n",
+      "                         'location': ['Patient', 'Line 1, Col 2'],\n",
+      "                         'severity': 'error'}],\n",
+      "              'resourceType': 'OperationOutcome',\n",
+      "              'text': {'div': '<div '\n",
+      "                              'xmlns=\"http://www.w3.org/1999/xhtml\"><h1>Operation '\n",
+      "                              'Outcome</h1><table border=\"0\"><tr><td '\n",
+      "                              'style=\"font-weight: '\n",
+      "                              'bold;\">ERROR</td><td>[Patient, Line 1, Col '\n",
+      "                              '2]</td><td><pre>Profile '\n",
+      "                              'http://fhir.kids-first.io/StructureDefinition/Patient, '\n",
+      "                              \"Element 'Patient.gender': minimum required = 1, \"\n",
+      "                              'but only found 0</pre></td>\\n'\n",
+      "                              '\\t\\t\\t\\t\\t\\n'\n",
+      "                              '\\t\\t\\t\\t\\n'\n",
+      "                              '\\t\\t\\t</tr>\\n'\n",
+      "                              '\\t\\t</table>\\n'\n",
+      "                              '\\t</div>',\n",
+      "                       'status': 'generated'}},\n",
+      " 'status_code': 412}\n",
+      "Validate http://hapi.fhir.org/baseR4/Patient/$validate, kidsfirst-test-patient\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Let's create an invalid resource and make sure it fails \n",
+    "# Invalid since its missing the REQUIRED gender attribute\n",
+    "patient_rs_invalid = {\n",
+    "    \"resourceType\":\"Patient\",\n",
+    "    \"id\": \"kidsfirst-test-patient\",\n",
+    "    \"text\": {\n",
+    "        \"status\": \"generated\",\n",
+    "        \"div\": \"<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\"><p>An invalid patient resource for testing purposes</p></div>\"\n",
+    "    },\n",
+    "    \"meta\": {\n",
+    "        \"profile\": [\"http://fhir.kids-first.io/StructureDefinition/Patient\"]\n",
+    "    }\n",
+    "}\n",
+    "validate_rs(patient_rs_invalid, expected_success=False, print_result=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### Add an Attribute"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "#### Extensions\n",
+    "In FHIR, when you want to add a new attribute to a `StructureDefinition`, its known as adding an \"extension\" since you are extending the definition of the base model.\n",
```

Comment:
```suggestion
    "Adding a new attribute to a `StructureDefinition` is known as adding an \"extension\", since you are extending the definition of the base model.\n",
```